### PR TITLE
[Side Modal]!: Align Side Modal footer actions to left

### PR DESCRIPTION
**Pull request by PixeledCode**
- updates the alignment of `SideModalFooterActions`
- updates the storybook

### CODE REVIEW

File: .changeset/smooth-ads-admire.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/side-modal": major
+"@twilio-paste/core": major
```
Comment (krisantrobus) [inline .changeset/smooth-ads-admire.md:3]: Check with @nkrantz but if it's not a code breaking change and style should this just be a minor?

```suggestion
"@twilio-paste/side-modal": minor
"@twilio-paste/core": minor
```

Comment (nkrantz) [reply .changeset/smooth-ads-admire.md:3]: yes minor

Comment (nkrantz) [reply .changeset/smooth-ads-admire.md:3]: also might be nice to have a bit more context in the changeset as to why we're making this change and what design action is needed

**Review by krisantrobus — DISMISSED** [review]

File: .changeset/smooth-ads-admire.md (hunk 1 above)
Comment (PixeledCode) [reply .changeset/smooth-ads-admire.md]: @nkrantz changeset updated

**Review by serifluous — APPROVED** [review]

**Review by nkrantz — APPROVED** [review]
👏 thanks for updating the changeset, this is so clear and helpful to anyone who sees it!

**Pull request merged**